### PR TITLE
Added Reference

### PR DESCRIPTION
**Pull request by Vipinofficial11**
Added reference.

### CODE REVIEW

**Comment by Vipinofficial11** [comment]
@itsankit-google  please review and merge.

File: .github/workflows/e2e.yml [hunk 1]
```diff
@@ -31,8 +31,7 @@ jobs:
       - uses: haya14busa/action-workflow_run-status@967ed83efa565c257675ed70cfe5231f062ddd94
       - uses: actions/checkout@v2.3.4
         with:
-          repository: data-integrations/servicenow-plugins
-          path: plugin
```
Comment (itsankit-google) [inline .github/workflows/e2e.yml]: Why do we need to remove these lines?

Comment (Vipinofficial11) [reply .github/workflows/e2e.yml:35]: We are using ref so we don't really need these lines.

Comment (itsankit-google) [reply .github/workflows/e2e.yml]: We still need to specify the `path` since it is being used by the `run_e2e_test` script

Comment (Vipinofficial11) [reply .github/workflows/e2e.yml]: Added.

File: .github/workflows/e2e.yml [hunk 2]
```diff
@@ -82,20 +82,18 @@ jobs:
...
         uses: Sibz/github-status-action@67af1f4042a5a790681aad83c44008ca6cfab83d
         if: always()
         with:
           authToken: ${{ secrets.AUTH_TOKEN }}
           state: success
           context: Cucumber report
-          target_url: https://data-integrations.github.io/servicenow-plugins/cucumber-reports/${{ github.ref }}/advanced-reports/cucumber-html-reports/overview-features.html
           sha: ${{github.event.pull_request.head.sha || github.sha}}
+
+      - name: Upload files to GCS
```
Comment (itsankit-google) [inline .github/workflows/e2e.yml]: nit: `files` -> `reports`

Comment (Vipinofficial11) [reply .github/workflows/e2e.yml]: updated

File: .github/workflows/e2e.yml [hunk 3]
```diff
@@ -82,20 +82,18 @@ jobs:
...
         uses: Sibz/github-status-action@67af1f4042a5a790681aad83c44008ca6cfab83d
         if: always()
         with:
           authToken: ${{ secrets.AUTH_TOKEN }}
           state: success
           context: Cucumber report
-          target_url: https://data-integrations.github.io/servicenow-plugins/cucumber-reports/${{ github.ref }}/advanced-reports/cucumber-html-reports/overview-features.html
           sha: ${{github.event.pull_request.head.sha || github.sha}}
+
+      - name: Upload reports to GCS
```
Comment (itsankit-google) [inline .github/workflows/e2e.yml]: It should be the second last step not the last.

Comment (Vipinofficial11) [reply .github/workflows/e2e.yml]: changed to second last step.

File: .github/workflows/e2e.yml [hunk 4]
```diff
@@ -82,13 +82,12 @@ jobs:
...
-        uses: peaceiris/actions-gh-pages@v3
-        if: always()
-        with:
-          github_token: ${{ secrets.AUTH_TOKEN }}
-          publish_dir: ./plugin/target/cucumber-reports
-          destination_dir: cucumber-reports/${{ github.ref }}
+     - name: Upload reports to GCS
+       uses: google-github-actions/upload-cloud-storage@v0
+       if: always()
+      with:
```
Comment (itsankit-google) [inline .github/workflows/e2e.yml]: indentation

Comment (Vipinofficial11) [reply .github/workflows/e2e.yml]: corrected.

**Review by itsankit-google — APPROVED** [review]

**Pull request merged**